src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  hasError: boolean,
+  errorMessage: string,
+  onClear: () => void
+};
+
+export const Notification: React.FC<Props> = ({
+  hasError,
+  errorMessage,
+  onClear,
+}) => {
+  useEffect(() => {
+    setInterval(onClear, 3000);
+  }, [errorMessage]);
```

Comment:
so, where do you clear the interval?

Reply:
check @GarryRocksman comment
`setInterval` creates the process that will call your function each 3 seconds for now